test/unit/presenters/formats/popular_links_format_presenter_test.rb

```diff
@@ -0,0 +1,33 @@
+require "test_helper"
+
+class PopularLinksFormatPresenterTest < ActiveSupport::TestCase
+  def subject
+    Formats::PopularLinksPresenter.new(FactoryBot.create(:popular_links))
+  end
+
+  context "#render_for_publishing_api" do
+    setup do
+      @result = subject.render_for_publishing_api
+    end
+
+    should "have expected required fields" do
```

Comment:
Should we consider making the tests here a bit more consistent with other format presenter tests? They all seem to have [individual tests for fields](https://github.com/alphagov/publisher/blob/cd76211dda73c9bb2dbc96d3c5bb126413cd6f9d/test/unit/presenters/formats/edition_format_presenter_test.rb#L144), with names like "[:change_note]".

Reply:
I did not notice that, I was bit iffy on it. I felt it was a bit of overkill to have separate tests for each field.I still feel we dont need to do that